postgrest.cabal

```diff
@@ -29,17 +29,17 @@ executable postgrest
     -rtsopts
     "-with-rtsopts=-N -I2"
   default-language:    Haskell2010
-  build-depends:       aeson (>= 0.8 && < 0.10) || (>= 0.11 && < 0.12)
+  build-depends:       aeson (>= 0.11 && < 0.12) || (>= 1.0 && < 1.1)
```

Comment:
The 0.10 series was actually known to be buggy so I'd like to skip it. But I could do something like

```
aeson (>= 0.8 && < 0.10) || (>= 0.11 && < 1.1)
```